src/condor_credd/condor_credmon_oauth/condor_vault_storer

```diff
@@ -211,4 +239,14 @@ but make sure no other job is using them."
     if [ $? != 0 ]; then
         fatal "Failed to store condor credentials for $SERVICE"
     fi
+    if $IS_POMS_ANALYSIS; then
+        # poms does not need these
+        rm -f $VTOKEN-$SERVICE
```

Comment:
Since the whole sandbox is cleaned up after submission, I'd rather leave the files there. Here's a famous case of what can happen with errant deletions: https://games.slashdot.org/story/07/12/06/1312254/EVE-Online-Patch-Makes-XP-Unbootable

Reply:
Also if $BTOKEN and $VTOKEN are based on $CONDOR_VAULT_STORER_ID then they will be unique per user and there's no worry about clashing between different users.